src/js/components/WinningNumberInputs.js

```diff
@@ -0,0 +1,32 @@
+import { CLASS_NAME } from '../constants/index.js';
+
+const $winningNumbers = document.querySelectorAll(
+    `input.${CLASS_NAME.WINNING_NUMBER}`
+);
+const $bonusNumber = document.querySelector(`input.${CLASS_NAME.BONUS_NUMBER}`);
+
+function WinningNumberInputs() {
+    const handleKeyUp = (e) => {
+        const { target: input } = e;
```

Comment:
```suggestion
const handleKeyUp = ({ target: input }) => {
```
개인의 취향이지만 이렇게 사용해도 괜찮겠네요!